x11-plugins/wmacpi/files/wmacpi-2.3-makefile.patch

```diff
@@ -6,7 +6,7 @@
  CC	:= gcc
 -CFLAGS	+= $(OPT) -Wall -W -g -ansi
 -LDFLAGS += $(OPT) -lX11 -ldockapp
-+CFLAGS	:= -Wall -W -g -ansi
++CFLAGS	?= -Wall -W -g -ansi
 +LIBS := -lX11 -ldockapp
```

Comment:
```suggestion
+LIBS += -lX11 -ldockapp
```
why? Ultimately, I'd like to make `LIBS` a user flag too, so users can have an expectation of fixing underlinking by just adding it to their `package.env`

Reply:
This was from the old patch, but thanks for pointing out, I'll keep an eye out for such things in the future